Meta/serenity.sh

```diff
@@ -203,7 +205,11 @@ pick_host_compiler() {
         return
     fi
 
-    die "Please make sure that GCC version 12, Clang version 13, or higher is installed."
+    if [ "$(uname -s)" = "Darwin" ]; then
+        die "Please make sure that Xcode 14.3, Homebrew Clang 13, or higher is installed."
```

Comment:
Homebrew clang 13 is unlikely to work :)

Reply:
Ah, let me test it and update the search list, then

Reply:
Updated the requirements to LLVM Clang 14.